src/utils/base64.js

```diff
@@ -7,6 +7,8 @@
  * @flow
  */
 
+import { Buffer } from './buffer';
+
```

Comment:
As I understand https://github.com/facebook/flow/issues/3723 was merged and will be available in the next release.
Does that mean you can just change this line to:
```js
import Buffer from 'buffer';
```
and everything will work?